File: specs/master/modules/snapshot_and_restore/repositories/get_repository/get_repository_request.ts

/**namespace:Modules.SnapshotAndRestore.Repositories.GetRepository */
interface get_repository_request extends request {
	/**ambiguous_origin*/
	MasterTimeout: time;
	/**ambiguous_origin*/
	Local: boolean;
	/**ambiguous_origin*/
	Source: string;
	/**ambiguous_origin*/
	FilterPath: string;
}

File: output/csharp/Modules/SnapshotAndRestore/Repositories/GetRepository/GetRepositoryRequest.cs
using Nest.CommonOptions;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Modules {

	public class GetRepositoryRequest : RequestBase {
		
		[DataMember(Name="local")]
		public bool Local { get; set; }


		[DataMember(Name="master_timeout")]
		public Time MasterTimeout { get; set; }

	}
}
